readthedocs/search/faceted_search.py

```diff
@@ -65,35 +66,55 @@ def __init__(self, query=None, filters=None, user=None, use_advanced_query=True,
         }
         super().__init__(query=query, filters=valid_filters, **kwargs)
 
-    def _get_text_query(self, *, query, fields, operator):
+    def _get_queries(self, *, query, fields):
```

Comment:
The usage of this method is more clear in https://github.com/readthedocs/readthedocs.org/pull/7569